python/test/test_query.py

```diff
@@ -0,0 +1,96 @@
+from timeplus import Stream, Query
+from timeplus.error import TimeplusAPIError
+
+
+def test_query_with_stream(test_environment, test_table):
+    # Test the Query object
+    query_list = Query(env=test_environment).list()
+    count = len(query_list)
+
+    query = (
+        Query(env=test_environment)
+        .sql(query=f"SELECT * FROM {test_table}")
+        .create()
+    )
+
+    query_list = Query(env=test_environment).list()
+    count1 = len(query_list)
+
+    assert count1 == count + 1
+
+    query_result = query.run()
```

Comment:
as it is stream query, it will never end, and you will not get data as there is no new ingest to the stream